core/templates/dev/head/pages/exploration-player-page/learner-experience/conversation-skin.directive.ts

```diff
@@ -828,13 +840,34 @@ angular.module('oppia').directive('conversationSkin', [
                 return;
               }
             }
+
+            if (!$scope.isInPreviewMode &&
+              ENABLE_SOLICIT_ANSWER_DETAILS_FEATURE) {
+              initLearnerAnswerInfoService(
+                $scope.explorationId, ExplorationEngineService.getState(),
+                answer, interactionRulesService);
+            }
+
             NumberAttemptsService.submitAttempt();
 
             $scope.answerIsBeingProcessed = true;
             hasInteractedAtLeastOnce = true;
 
             PlayerTranscriptService.addNewInput(answer, false);
 
+            if ($scope.canAskLearnerForAnswerInfo()) {
+              $timeout(function() {
+                PlayerTranscriptService.addNewResponse(
+                  '<p translate="I18N_SOLICIT_ANSWER_DETAILS_QUESTION"></p>');
```

Comment:
No, don't put HTML in JS. Please find a way to do this that does not involve using HTML tags in the JS.

Reply:
Done